board/app/controllers/favorite_controller.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+class FavoriteController extends AppController 
+{
+
+ public function set_favorite()
```

Comment:
Done po. Thank you :bow:
--> https://github.com/maragatchalian/FoodFacts/commit/5643af557050eac86f3350ff6d2bb2bedc579c53